src/transformer/base/base.ts

```diff
@@ -42,13 +42,17 @@ function visitNode(node: ts.Node, visitor: Visitor, customFunctions: CustomFunct
     return node;
   }
 
-  const signature: ts.Signature = TypescriptHelper.getSignatureOfCallExpression(node);
+  const signature: ts.Signature | undefined = TypescriptHelper.getSignatureOfCallExpression(node);
 
-  if (!isFunctionFromThisLibrary(signature, customFunctions)) {
+  if (!signature || !isFunctionFromThisLibrary(signature, customFunctions)) {
     return node;
   }
 
-  const nodeToMock: ts.TypeNode = node.typeArguments[0];
+  const nodeToMock: ts.TypeNode | undefined = node.typeArguments?.[0];
+
+  if (!nodeToMock) {
```

Comment:
Again. this would mean calling createMock without a generic. This means disabling the typecheck. If consumers are willing to use it we should throw a more specific error :) Something like. Hei, it looks like you are using {whatever} without a generic? check the documentation at this link